libs/common-services/src/main/java/uk/gov/di/ipv/core/library/service/ConfigService.java

```diff
@@ -261,11 +261,8 @@ public String getActiveConnection(String credentialIssuerId) {
     }
 
     public String getComponentId(String credentialIssuerId) {
-        String activeConnection = getActiveConnection(credentialIssuerId);
-        final String pathTemplate =
-                ConfigurationVariable.CREDENTIAL_ISSUERS.getPath()
-                        + "/%s/connections/%s/componentId";
-        return getSsmParameterWithOverride(pathTemplate, credentialIssuerId, activeConnection);
+        var criConfig = getOauthCriActiveConnectionConfig(credentialIssuerId);
+        return criConfig.getComponentId();
     }
```

Comment:
We could consider removing this method entirely and inline this logic, but this is easier :)